components/Layout.tsx

```diff
@@ -152,10 +152,25 @@ const MainNavigation = () => {
   useEffect(() => {
     const icon = theme === 'dark' ? 'herobtn' : '';
     const menu = theme === 'dark' ? 'bg-white' : 'bg-black';
+    const dataTheme = theme === 'dark' ? 'dark' : 'light';
+    const keyShadow =
+      theme === 'dark'
```

Comment:
Why not defining the styles as classes in the global css? That approach seems a cleaner way to manage styles.

Reply:
@benjagm did the changes you told, is it okay now? Please review.